githubapp/ldap.py

```diff
@@ -47,7 +50,11 @@ def get_group_members(self, group_name):
             if entry['type'] == 'searchResEntry':
                 for member in entry['attributes'][self.LDAP_GROUP_MEMBER_ATTRIBUTE]:
                     try:
-                        member_list.append(self.get_attr_by_dn(member))
+                        if 'uid' in member:
+                            member_dn = member
+                        else:
+                            member_dn = f'uid={member},{self.LDAP_USER_BASE_DN}'
+                        member_list.append(self.get_attr_by_dn(member_dn))
```

Comment:
this breaks Active Directory lookups... probably need to break out a new method that handles the distinguishing of LDAP entry types 🤔